roles/ceph-mon/tasks/create_mds_filesystems.yml

```diff
@@ -15,3 +15,18 @@
   command: ceph --cluster {{ cluster }} fs new {{ cephfs }} {{ cephfs_metadata }} {{ cephfs_data }}
   changed_when: false
   when: ceph_release_num.{{ ceph_release }} > ceph_release_num.firefly
+
+- name: allow multimds
+  command: ceph --cluster {{ cluster }} fs set {{ cephfs }} allow_multimds true --yes-i-really-mean-it
+  changed_when: false
+  when:
+  - "{{ ceph_version | version_compare('10.1.2', '>=') }}"
```

Comment:
as per https://github.com/ceph/ceph-ansible/pull/953
this needs to be: `ceph_release_num.{{ ceph_release }} >= ceph_release_num.jewel`